### PR TITLE
[Merged by Bors] - chore: update version to pre-release 2

### PR DESCRIPTION
**Pull request by EstebanBorai**

### CODE REVIEW

**Review by nacardin — APPROVED** [review]

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.10.4-dev-1
\ No newline at end of file
+0.10.4-dev-2
```
Comment (morenol) [inline VERSION:1]: I think that we need to strip the `\n`

Comment (morenol) [reply VERSION:1]: but not sure if that issue was fixed

Comment (nacardin) [reply VERSION:1]: we should keep the newline

Comment (nacardin) [reply VERSION:1]: we should keep the newline

**Review by nacardin — CHANGES_REQUESTED** [review]

File: VERSION (hunk 1 above)
Comment (morenol) [reply VERSION:1]: In the past it had compilation errors when we added it, if it works now with the newline I am ok with it

**Review by nacardin — APPROVED** [review]

**Review by morenol — APPROVED** [review]

**Pull request closed without merging**